scaletest/dashboard/run.go

```diff
@@ -33,6 +43,16 @@ func NewRunner(client *codersdk.Client, metrics Metrics, cfg Config) *Runner {
 }
 
 func (r *Runner) Run(ctx context.Context, _ string, _ io.Writer) error {
+	err := r._run(ctx)
+	// If the context deadline exceeded, don't return an error.
+	// This just means the test finished.
+	if err == nil || errors.Is(err, context.DeadlineExceeded) {
+		return nil
+	}
+	return err
+}
+
+func (r *Runner) _run(ctx context.Context) error {
```

Comment:
I think we can ditch the underscore here?

```suggestion
func (r *Runner) run(ctx context.Context) error {
```

Reply:
Theres's a linter that complains about it. I can rename to `runUntilDeadlineExceeded` if you prefer though!